cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/BasicPullStrategy.java

```diff
@@ -85,10 +83,16 @@ public BasicPullStrategy(PullReplication pullReplication) {
     }
 
     public BasicPullStrategy(PullReplication pullReplication,
+                             PullConfiguration config) {
+        this(pullReplication, null, config);
+    }
+
+    public BasicPullStrategy(PullReplication pullReplication,
                              ExecutorService executorService,
                              PullConfiguration config) {
         Preconditions.checkNotNull(pullReplication, "PullReplication must not be null.");
 
+        // TODO pass this executor service down to the GetRevisionTask
```

Comment:
I assume this TODO and the associated TODOs in the `GetRevisionTaskThreaded` for executor configuration are going to be covered in a future PR?